packages/@guardian/source-react-components-development-kitchen/components/logo/Logo.tsx

```diff
@@ -117,25 +117,26 @@ const DecideLogo = ({ logoType }: { logoType: LogoType }) => {
 	}
 };
 
+const logoStyles = css`
+	/* Adding block display here so that the focus halo correctly covers the content */
+	display: block;
+	color: ${brandText.primary};
+	:hover {
+		color: ${brandText.primary};
+	}
```

Comment:
If there’s no text (it’s `visuallyHidden`), is it worth setting a colour at all?

Reply:
Good spot! I just copied the styling that was already there. But makes sense to remove it.